### ISSUE TITLE
Smart space empty

### ISSUE THREAD

**Issue by dxcore35** [Make-md/makemd-preview#2]
I have 10 items with class: project

But when creating SMART SPACE there is noting

![Screenshot 2023-04-03 at 10 39 33](https://user-images.githubusercontent.com/7279156/229443185-fce73c82-166d-484d-a3f8-31bb14bcb65d.jpg)

v0.7.4

**Comment by jp-cen** [comment]
Is it also empty when you expand the Space in the sidebar?

**Comment by dxcore35** [comment]
Yes. I tried to click on every item.... there is nothing expanding.

**Comment by jp-cen** [comment]
Can you confirm that the cases are correct in your notes (both the property and the value) and whether there is any errors. Also, can you check whether you see the line "make.md files Indexer" in your logs which indicates that the metadata has been parsed correctly.

**Comment by dxcore35** [comment]
- cases are correct
- I even CONTAINS instead of IS

I cannot located the make.md files indexer

**Comment by jp-cen** [comment]
Seems like there’s an issue with the indexer which runs all the queries, It’s either stuck or errored out, here’s a few questions that would helpm narrow it down.

1. Do you see any errors in the logs?
2. What size is your vault?
3. Is there a sync message in your Spaces?

Thanks!
